CONTRIBUTING.md

```diff
@@ -109,7 +109,7 @@ These are unstable/unsupported platforms, and in all likelihood, whatever the is
     *   Examples: Windows 8.1 32-bit, Linux Mint 17.1 64-bit, Windows 10 Fall creators Update 64-bit, etc.
 
 
-*   **Report only if you can reproduce the issue with an official stable release, a beta release, or with the most recent upstream changes (in this last case specify the specific commit you are on).** (beta testing is encouraged :smile:). We do not porvide support for bugs with [PortableApps][portableapps-url] builds.
```

Comment:
you forgot to remove portableapps-url below.

Reply:
@Chocobo1 Thanks, done